snyk.md

```diff
@@ -1,4 +1,6 @@
-# Snyk
+# Snyk 
+
+[Deprecated] - We recommend using [Github Dependency Graph](https://docs.github.com/en/code-security/supply-chain-security/understanding-your-software-supply-chain/about-the-dependency-graph) and [Dependabot Alerts](https://docs.github.com/en/code-security/dependabot/dependabot-alerts) to analyse dependencies for vulnerabilities.
```

Comment:
Could emphasise this information visually? Maybe using a [warning](https://github.com/orgs/community/discussions/16925), or making it bigger?